pkgs/tools/networking/termshark/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, go }:
+
+stdenv.mkDerivation rec {
+  name = "termshark-${version}";
```

Comment:
use pname = "termshark" instead

Reply:
Could you explain to me what the difference between `name` and `pname` is?

(Feel free to point me to the documentation. I could not find it ;-) .)

Reply:
I invite you to grep  pkgs/stdenv/generic/make-derivation.nix for pname and you will see that name = pname + version if name is not set. It's in general more elegant to set pname instead of name except if you need to use name later in the derivation, in the url for instance.